pytensor/tensor/basic.py

```diff
@@ -14,6 +14,7 @@
 from typing import cast as type_cast
 
 import numpy as np
+import torch
```

Comment:
This line is making torch non optional which is probably why the build wheels is failing